app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
The Dwarf class did not implement any abstract function, but it has to in this case.
But In the task this class has to be abstraction too.
Something like this: 
 ```python
 class Player(ABC):
    @abstractmethod
    def some_method_from_abstract()
         pass
class Dwarf(Player, ABC)
   def_some_method

class DwarfBlacksmith(Dwarf):
    def some_method_from_abstract()
```